.github/workflows/lints.yaml

```diff
@@ -0,0 +1,14 @@
+name: Lints
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    name: Lint for style issues
+    runs-on: ubuntu-latest
```

Comment:
This is specifically about linting the commit message right? If so, might be helpful to amend the name here and the commit subject to make that clear (else it gives the impression that this is essentially doing cargo fmt)